Removes marines

comma opened the pull request:
:cl: Chinsky
rscdel: After yet another PR fiasco over field executions, Marine contingent is no longer allowed on Torch. Those not under active investigation are offered a transfer to Expeditionary Corps if they want to participate in the mission.
/:cl:
Leaves clothing, items, etc untouched.
Only removes data structures needed to spawn as them and such.
Removes Military Police alt title cause requested and well, they're not military anymore.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

rainbowEscapist commented:
No.

ThatOneGuy42 commented:
If you don’t include NT with this it isn’t good enough

ThatOneGuy42 commented:
Also you fucking madman

ghost commented:
I'd suggest creating some kind of an escort ship for Torch where Fleet and Marines would be.

HeyBanditoz commented:
It begins.

On December 25, 2017 10:42:02 PM PST, Chinsky <notifications@github.com> wrote:
>Feels good man
>Open to suggestions on where to stick their swag instead of removing, I
>mean it's a lot of spritework.
>Mostly opening to get a scale of it and to remind that this is coming.
>
><!-- 
>Do not forget to add a changelog when you have made admin/player facing
>changes that can alter gameplay.
>Examples which require a changelog entry include:
>* Adding/removing objects that players may interact with, or the way
>they function.
>* Adding/removing/altering admin tools.
>* Changing the map.
>
>Examples were changelog entries are optional/not typically required:
>* Cosmetic changes such as descriptions, sound effects, etc.
>* Optimizations and other changes to underlying systems which do not
>affect gameplay.
>* Minor bug fixes.
>
>You find a README and example file in .\html\changelogs\ for further
>instructions.
>-->
>
>You can view, comment on, or merge this pull request online at:
>
>  https://github.com/Baystation12/Baystation12/pull/19932
>
>-- Commit Summary --
>
>  * Removes fleet and marines
>
>-- File Changes --
>
>    M maps/torch/datums/uniforms.dm (53)
>    D maps/torch/datums/uniforms_fleet.dm (156)
>    D maps/torch/datums/uniforms_marine.dm (199)
>    M maps/torch/items/clothing/solgov-accessory.dm (289)
>    M maps/torch/items/clothing/solgov-head.dm (126)
>    M maps/torch/items/clothing/solgov-suit.dm (130)
>    M maps/torch/items/clothing/solgov-under.dm (172)
>    M maps/torch/items/items.dm (16)
>    M maps/torch/job/jobs.dm (247)
>    M maps/torch/loadout/loadout_head.dm (12)
>    M maps/torch/torch_ranks.dm (467)
>
>-- Patch Links --
>
>https://github.com/Baystation12/Baystation12/pull/19932.patch
>https://github.com/Baystation12/Baystation12/pull/19932.diff
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/Baystation12/Baystation12/pull/19932

Chiquiii commented:
About time, the EC prevail once again.

AtomicWorm commented:
But why?

Minijar commented:
No thank you. I have actual characters in these.

Hubblenaut commented:
Oh boy I'm going to miss them. Personally I'd like at least the fleet to stay.

Did you remove the MP alt title along with this? Difficult to check on mobile.

comma commented:
>But why?

We've had approximately billion polls on this matter, tl;dr whichever way it goes military is not going to be around.

>Did you remove the MP alt title along with this?

Left it for now

Spamcat commented:
**JUST DO IT**

Zero-Bits commented:
This effects my characters by making them happier. 0/10

EcklesFire commented:
Well if you're gonna remove fleet and marines I'm assuming/hoping you're making Research EC as well to bring the ship in line with EC supreme. 
Otherwise, it just sorta looks like shafting one half while keeping the other.

All in all I believe that this PR should generally be bigger in scope and redo the manifest as a whole to make it into an EC only ship, if it even gets to that point.

Personally, I really don't want this, I thought we were actually, finally, in a fairly decent spot and people were just...Enjoying themselves?

hey-it-stev commented:
If we remove NT I hope we get a lot more contractor slots to make up for the massive loss of civilian roles.

ghost commented:
This remains open for more than a hour, but the handholding PR is closed by the OP himself the moment it was opened. Wew.

AtomicWorm commented:
So what about the people who have Fleet and marine characters? You'll just basically tell them to get fucked?

comma commented:
>So what about the people who have Fleet and marine characters? You'll just basically tell them to get fucked?

I guess I will.

Kelenius commented:
>So what about the people who have geneticist characters? You'll just basically tell them to get fucked?

>So what about the people who have Exodus characters? You'll just basically tell them to get fucked?

>So what about the people who have Resomii characters? You'll just basically tell them to get fucked?

ivygofly commented:
Maybe finish Torch 2.0 first, then dismantle Torch 1.0, kay? 
It''s coming anyway, let the people who enjoy it live it out in peace.

I mean, no one started gutting Exodus after the shift to Torch was made official?

comma commented:
Whatever 'torch 2.0' will be, it'll be different thing at some point in bright communisitc future.
This is a change to existing thing, one that had been mulled over in countless threads, and had not been implemented only because no one got off their ass to do so.
Here's last couple.

https://baystation12.net/forums/threads/setting-change-more-milrp-or-less-milrp.5713/
https://baystation12.net/forums/threads/conditional-vote.5740/

Zero-Bits commented:
>EC Supreme gets least votes in setting change poll
>EC Supreme anyway

That said, the results of this PR aren't really meaningful, considering the fact that Torch 2.0 is coming a lot sooner than you'd think.

babydoll commented:
We have no evidence of that as it is inexplicably happening behind closed doors

comma commented:
They'll be gone in that great bright Torch 2.0 future anyway, so I don't really see how it's relevant to this PR.

Zero-Bits commented:
Yeah, chinsky has a point. This should be considered entirely separate from 2.0. The results here don't mean anything for 2.0, and honestly, I'm ambivalent about them.

As for people who have characters affected by change [x], [y] or [z]? That sucks. Characters are neat. Being able to play them is neat. But if you can't find a way to fit them into a new setting. Then oh well. We can't cater to everyone. I'll lose characters too, and that sucks. But oh well.

AtomicWorm commented:
How funny that the people supporting this here are those who only play NT. Remove NT as well. Let's see how much crying that'll get you.

BlueNexus commented:
Contentious label please.

Kelenius commented:
>How funny that the people supporting this here are those who only play NT.

How funny that this is bullshit.

Chiquiii commented:
> Contentious label please.

We don't need labels where we're going.

Anton-Kr commented:
Fuck no.

SparklySheep commented:
> How funny that the people supporting this here are those who only play NT. Remove NT as well. Let's see how much crying that'll get you.

Oh man don't get my hopes up even more.

Anton-Kr commented:
Remove SEA office as well then. (Or just close this PR)

comma commented:
Good point, but I'd rather tangle with the map conflicts after I have clear statement on if this goes in or not.

Crushtoe commented:
>making up labels

You're just making it more annoying to sort through joke labels on issues.

OhEmGee commented:
that's a spicy meatball-a

HeyBanditoz commented:
What becomes of the SEA's office?
EDIT: I'm a dumbass.

Anton-Kr commented:
Before this meme continues coild we have a forum thread pls

hey-it-stev commented:
We've already had many, many forum threads. In fact they've been cited elsewhere on this github thread.

Anton-Kr commented:
And the results were inconclusive at best. 

Half baked mil-rp is dumb. Can't we just wait until we have the fully civilian Torch. Contractor slots were added to pretty much every possible dept besides security to keep civilian players happy. All this will do is restrict military players to now be part of the EC and remove any kind of flavor or variety for those of us who enjoy playing military characters. Regardless Ill just keep playing my characters in the same way with the pretense that they tramsferred to the EC for the duration of the mission in the same way that I played them. It just seems like a meaningless feature loss.

Sunbeamstress commented:
God Chinsky you're my hero.

Also I don't play NT ever, where is your god now

ThatOneGuy42 commented:
Either remove NT with this and fill in some more contractors or I don’t think this should be pushed. Those are my two cents

comma commented:
I'd rather not touch NT.
More contractors - where?

meyar commented:
You madman. +1.

PurpleMartinJCK commented:
This is a lore based change. If it's gets approved by the lore peeps then it can be reviewed.

Hubblenaut commented:
If this happens, just please, for the love of god, take the military police alt title along with them.

ThatOneGuy42 commented:
I’m alright with Marines being removed but not Fleet.

comma committed:
Removes marines from Torch

Gear etc still there, just removed the data structured used for spawning as jobs.

Removed MP alt title since MAA is EC+Fleet.

comma commented:
Alright, fleet left untouched.

comma commented:
Apparently this doesn't count as 'lore approval'? @ThatOneGuy42 @NobleCaos  pls look now.

Techhead0 reviewed (approved):
Remember that time when I ~~threatened~~ offered to unrevert #17428 if staff didn't come up with anything in a few weeks?

ThatOneGuy42 commented:
I approve of this change.

PurpleMartinJCK commented:
@ThatOneGuy42 why's that?

meyar commented:
Helps focus things, for one. One less faction to make things feel schizophrenic. Helps kabosh the real or perceived encouragement of the "hoorah pewpew power fantasy because the depth of my understanding of marines comes from gung ho western media" that we tried to stamp out for years on the exodus with the deluge of ex-marine space nam shite (and the torch even came with its own space nam: gaia) 

Let's is focus more on the differences between the military  (IE fleet) and non-miltary  (IE EC) and civilian personnel. 

The marines always felt tacked on.

PurpleMartinJCK commented:
@Techhead0 him saying he approves of the change doesn't mean it's lore approved. If he'd like to push this through he can come on and say that he, as headmin, is approving the functionality.

PurpleMartinJCK commented:
I got clarification from ToG that he means he approves the gameplay of the PR as a whole, as headmin.

meyar commented:
I await the 9000 polls that will follow.

ghost commented:
Polls don't mean anything, except as a visual indicator for what someone feels is the more "me-aligned" button.

meyar commented:
I think we should have a vote on whether or not votes mean anything /s

Minijar commented:
Fuck the playerbase again eh?
